### ISSUE TITLE
Participant boilerplate comparison

### ISSUE THREAD

**Issue by manuelfuenmayor** [metanorma/metanorma-ieee#105]
As per https://github.com/metanorma/metanorma-ieee/issues/101#issuecomment-1185625112

@ronaldtse I see four documents on [mn-samples-ieee-private](https://github.com/metanorma/mn-samples-ieee-private):

For master-1-standard:
![master-1](https://user-images.githubusercontent.com/33627611/179255992-63aa8bf3-5c2b-44af-9cc3-351495956b5c.PNG)

For master-2-standard:
![master-2](https://user-images.githubusercontent.com/33627611/179256043-59d1da28-29db-4959-8edb-65bb89edfbd9.PNG)

For P2830_d1:
![master-3](https://user-images.githubusercontent.com/33627611/179257402-57b1432c-b990-486e-875a-54c5a03f3857.PNG)

For 2830-2021:
![master-4](https://user-images.githubusercontent.com/33627611/179257659-99825006-7af3-4f9b-ad32-46193e5fa20d.PNG)

The first one is a guide, and the other ones are standards.
Maybe the difference resides on the stage of the document?

**Comment by opoudjis** [comment]
We are allowing all the text to be user-supplied, other than the initial sentence of each participants subsection. @manuelfuenmayor @ronaldtse If we must make that be user-supplied as well, we will, but I'd really rather avoid that. Currently we are using the wording of master-1-standard

**Comment by ronaldtse** [comment]
> If we must make that be user-supplied as well, we will, but I'd really rather avoid that. 

I don't think we need to make it user-supplied. I will seek IEEE clarification.

**Comment by ronaldtse** [comment]
* The sentence in `master-1-standard` uses an identical pattern as `2830-2021`, differing at the document type.
* `P2830_d1` and `2830-2021` sentences differ where the word "IEEE" is substituted as "draft" for the draft.

I think these are the only two things we need to handle.

**Comment by Intelligent2013** [comment]
I've just generated PDF for `SWF-Corpus_and_IEEEP2874\sources\ieee-p2874\p2874.adoc`:

<img width="760" height="352" alt="Image" src="https://github.com/user-attachments/assets/c1602fb0-8662-499f-bd95-1401af788b7a" />

and for `mn-samples-ieee-private\sources\2830\document.adoc`:

<img width="671" height="315" alt="Image" src="https://github.com/user-attachments/assets/a40d1435-1455-4f1c-9164-499f797e40a7" />

First is `draft` layout, second is the `standard` layout in the XSLT. I don't remember why different styles are using.
Ok, I'll do without space.

**Comment by opoudjis** [comment]
> First is draft layout, second is the standard layout in the XSLT. I don't remember why different styles are using.

From the initial ticket, it looks like we were trying to reconstruct the difference in spacing between documents we found, as a difference in draft versus published status. Given the formatting of the Word document, I'm pretty sure this was just an editorial error.
